microshift_release_notes/microshift-4-15-release-notes.adoc

```diff
@@ -197,7 +197,7 @@ In the following tables, features are marked with the following statuses:
 [id="microshift-4-15-asynchronous-errata-updates"]
 == Asynchronous errata updates
 
-Security, bug fix, and enhancement updates for {microshift-short} {product-version} are released as asynchronous errata through the Red Hat Network. All {microshift-short} {product-version} errata are https://access.redhat.com/downloads/content/290/[available on the Red Hat Customer Portal]. Read the https://access.redhat.com/support/policy/updates/openshift[{microshift-short} Life Cycle] for more information about asynchronous errata.
+Security, bug fix, and enhancement updates for {microshift-short} {product-version} are released as asynchronous errata through the Red Hat Network. All {microshift-short} {product-version} errata are https://access.redhat.com/downloads/content/290/[available on the Red Hat Customer Portal]. Read the https://access.redhat.com/product-life-cycles?product=Red%20Hat%20build%20of%20Microshift,Red%20Hat%20Device%20Edge[{microshift-short} Life Cycle] for more information about asynchronous errata.
```

Comment:
Move the intro statement to the front of the sentence to let readers know if they are interested in the link: "For more information about asynchronous errata, read the https://access.redhat.com/product-life-cycles?product=Red%20Hat%20build%20of%20Microshift,Red%20Hat%20Device%20Edge[{microshift-short} Life Cycle]."

Reply:
Done